.eslintrc

```diff
@@ -1,12 +1,24 @@
 {
-  "extends": "airbnb-base/legacy",
```

Comment:
We may want to add some overrides to their rules for our application as well. A couple examples:

- it throws errors for unnecessary quoted property names. Do we want to consistently quote them since there are so many hyphenated properties in css?

- It throws errors for missing spaces before function parentheses, but it looks like in `styled-components` we aren't enforcing that.